src/fsharp/symbols/Exprs.fs

```diff
@@ -197,6 +197,62 @@ module FSharpExprConvert =
         | Expr.Op(TOp.ValFieldSet rfref, _, _, _)  when IsStaticInitializationField rfref -> Some ()
         | _ -> None
 
+    let (|ILUnaryOp|_|) e = 
+        match e with 
+        | AI_neg -> Some mkCallUnaryNegOperator
+        | AI_not -> Some mkCallUnaryNotOperator
+        | _ -> None
+
+    let (|ILBinaryOp|_|) e = 
+        match e with 
+        | AI_add
+        | AI_add_ovf
```

Comment:
Shouldn't that ``ovf`` ones make a call to the Checked version of the addition operator?

Note that the inversion is not unique - e.g. checked addition on float is the same opcode as unchecked addition on float (there being no checking)., I don't think this matters for your purposes.